pkg/networkservice/common/timeout/server.go

```diff
@@ -51,7 +50,7 @@ func NewServer(ctx context.Context) networkservice.NetworkServiceServer {
 }
 
 func (t *timeoutServer) Request(ctx context.Context, request *networkservice.NetworkServiceRequest) (*networkservice.Connection, error) {
-	logEntry := logger.Log(ctx).WithField("timeoutServer", "request")
+	logEntry := log.FromContext(ctx).WithField("timeoutServer", "request")
```

Comment:
I believe such places should be renamed from `logEntry` to `logger` or something like that. Because earlier there was:
```go
logEntry := log.Entry(ctx).WithField(...)
```
and now it doesn't make any sense.